### PR TITLE
Corkyw10/about updates

### PR DESCRIPTION
**Pull request by corkyw10**
# About page updated

- Added Marc Solé and Roel to the about page
- Moved Marc Garcia to former members
- Renamed OSVF to Embodied AI Foundation

### CODE REVIEW

File: _team/developers/02-bernat.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name_: Bernat Pina
 description: Software engineer
```
Comment (bernatx) [inline _team/developers/02-bernat.md:3]: Not sure if German wants to show here that he is in a **Jr** position.

File: _team/developers/01-roel.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name_: Roel Algaba Brizuela
+description: Software engineer
```
Comment (bernatx) [inline _team/developers/01-roel.md:3]: Not sure if German wants to show here that he is in a **Jr** position.

**Review by bernatx — CHANGES_REQUESTED** [review]
Maybe Roel needs to be noted that is in a **Jr** position?

File: _team/developers/01-roel.md (hunk 2 above)
Comment (corkyw10) [reply _team/developers/01-roel.md:3]: I spoke with German, Software Engineer is fine

**Pull request merged**